vsts-extension-code-push.json

```diff
@@ -57,7 +62,55 @@
             "type": "ms.vss-distributed-task.task",
             "targets": [ "ms.vss-distributed-task.tasks" ],
             "properties": {
-                "name": "code-push-vsts-task"
+                "name": "Tasks/code-push-vsts-task"
+            }
+        },
+        {
+            "id": "codepush-service-endpoint",
+            "description": "CodePushAuthenticationEndpoint",
+            "type": "ms.vss-endpoint.service-endpoint-type",
+            "targets": [
+                "ms.vss-endpoint.endpoint-types"
+            ],
+            "properties": {
+                "name": "codepush-auth-key",
+                "displayName": "Code Push",
+                "url": "",
+                "inputDescriptors": [],
+                "authenticationSchemes": [
+                    {
+                        "type": "ms.vss-endpoint.endpoint-auth-scheme-basic",
+                        "inputDescriptors": [
+                            {
+                                "id": "username",
```

Comment:
I can try to change this to an endpoint-auth-scheme-token and test it. That should prevent us having to define and hide the username field.